src/furtherWork/petC.js

```diff
@@ -0,0 +1,96 @@
+// Class Syntax Pet
```

Comment:
This comment should be removed

See the comments in `petTrack.js`, they all apply in here too